src/server/models/Volunteer.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    Volunteer,
+  } from '@/types/volunteer';
+  import { Model, Schema, model, models } from 'mongoose';
+  
+  const VolunteerSchema = new Schema(
+    {
+      firstName: {
+        type: String,
+        required: true,
+      },
+      lastName: {
+        type: String,
+        required: true,
+      },
+      userName: {
+        type: String,
+        required: true,
+      },
+      id: {
+        type: Number,
+        required: true,
+      },
+      email: {
+        type: String,
+        required: true,
+      },
+      phoneNumber: {
+        type: Number,
+        required: true,
+      },
+      events: {
+        type: Schema.Types.ObjectId,
```

Comment:
the type should be [Schema.Types.ObjectId] with the brackets to indicate that it is an array.